README.md

```diff
@@ -30,36 +30,48 @@ equivalent to `createMethod().bind(obj)`.
 If the operator’s right-hand side does not evaluate to a function during runtime,
 then the program throws a `TypeError`.
 
-Function binding has equal **[precedence][]** with
-**member expressions**, call expressions, `new` expressions with arguments,
-and optional chains.
-
-[precedence]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence
-
-| Left-hand side                  | Example       |
-| ------------------------------- | ------------- |
-| Primary expressions             | `a->fn`       |
-| Member expressions              | `a.b->fn`     |
-| Call expressions                | `a()->fn`     |
-|`new` expressions with arguments | `new C()->fn` |
-| Optional chains                 | `a?.b->fn`    |
-
-The bound functions created by the bind operator
+The bound functions created by the bind-`this` operator
 are **indistinguishable** from the bound functions
 that are already created by [`Function.prototype.bind`][bind].
 Both are **exotic objects** that do not have a `prototype` property,
 and which may be called like any typical function.
 
-Similarly to the `?.` optional-chaining token,
-the `->` token may be **padded by whitespace**.\
+From this definition, `o->f(...args)`
+is **indistinguishable** from `Function.prototype.call(o, ...args)`,
```

Comment:
```suggestion
is **indistinguishable** from `Function.prototype.call(f, o, ...args)`,
```

Reply:
technically this isn't accurate either - you need `f.call(o, ...args)`, because if you just call `Function.prototype.call` directly then its receiver is `Function.prototype`.